src/test/ui/operator-recovery/arithmetic-right-shift.stderr

```diff
@@ -0,0 +1,8 @@
+error: invalid shift operator `>>>`
+  --> $DIR/arithmetic-right-shift.rs:2:22
+   |
+LL |     println!("{}", 1 >>> 2);
+   |                      ^^^ `>>>` is not a valid right shift operator, consider casting to a signed integer and right shifting normally.
```

Comment:
I've never used `>>>` so I might be confused, but isn't `>>>` usually a logical shift (that brings in zeros), thus one should cast to an **un**signed integer?

That's what it seems to be in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Unsigned_right_shift and https://docs.oracle.com/javase/specs/jls/se7/html/jls-15.html#jls-15.19, at least.

Reply:
Yoooou are correct, I mixed the two up in my head. I'll fix it in a few minutes.